sdk/analytics/src/main/java/com/klaviyo/analytics/model/Profile.kt

```diff
@@ -12,28 +12,28 @@ class Profile(properties: Map<ProfileKey, Serializable>?) :
 
     fun setExternalId(identifier: String) = apply { this.externalId = identifier }
     var externalId: String?
-        get() = (this[ProfileKey.EXTERNAL_ID]) as String?
+        get() = this[ProfileKey.EXTERNAL_ID] as? String
```

Comment:
Hmm would this have failed if you passed in a non string before?

Reply:
```
print(3 as String?)
> java.lang.ClassCastException: class java.lang.Integer cannot be cast to class java.lang.String (java.lang.Integer and java.lang.String are in module java.base of loader 'bootstrap')
	at Line_1.<init>(Line_1.kts:1)

print(3 as? String)
> null
```

Reply:
yeah looks like it would have raised